docs/citing.rst

```diff
@@ -0,0 +1,33 @@
+Citation Guidelines
+===================
+
+When publishing work that uses pysatNASA, please cite the package and
+any package it depends on that plays an important role in your analysis.
+Specifying which version of pysatNASA used will also improve the
+reproducibility of your presented results.
+
+pysatNASA
+---------
+
+* Klenzing, J., et al. (2020).
+  pysat/pysatNASA: Initial Release (Version 0.0.1). Zenodo.
+  doi:10.5281/zenodo.3986132
+
+.. code-block:: latex
+
+  @software{jeff_klenzing_2020_3986132,
+    author       = {Jeff Klenzing and
+                    Russell Stoneback and
+                    Angeline Burrell and
+                    Asher Pembroke and
+                    Matthew Depew and
+                    Carey Spence and
+                    Asher Pembroke},
```

Comment:
Isn't the name supposed to be `Last, First and`?

Reply:
copied this directly from zenodo, but happy to change to keep in style

Reply:
Yeah, this is something that will break BibTeX, regardless of what zenodo says 🐱